jsapp/js/components/map.es6

```diff
@@ -27,46 +27,51 @@ import {
   QUERY_LIMIT_DEFAULT,
 } from '../constants';
```

Comment:
Sorry in advance for `map.es6`. Best to review these commits separately.

- 🤖 Prettier and ESLint autofixes [here](https://github.com/kobotoolbox/kpi/pull/4505/commits/be3a9b6041e451b02c671220be50b5ca33cae1a8)
- ✋ Manual ESLint fixes [here](https://github.com/kobotoolbox/kpi/pull/4505/commits/16f242bd05aafa5b6ae8bb5d31670ccb0e568aa1)
	- Manually hoist variables to make scope clearer and use 'let'
	- Carefully replace 8 loose comparisons with strict ones (`===`, `!==`)
		- [https://dorey.github.io/JavaScript-Equality-Table/](https://dorey.github.io/JavaScript-Equality-Table/) was useful here
- ✋ Fix map screen crash [here](https://github.com/kobotoolbox/kpi/pull/4505/commits/bcdbaf0a42c2576446e7f48c873e68e981a209ef)